Source/DynamicFeatureRegistry.swift

```diff
@@ -0,0 +1,46 @@
+// Copyright 2019, Oath Inc.
+// Licensed under the terms of the MIT license. See LICENSE file in https://github.com/yahoo/Override for terms.
+
+import Foundation
+
+/// DynamicFeatureRegistry a subclass of FeatureRegistry which provides an interface for using
+/// a collection of dynamic features instead of just pre-defined ones. When a feature is added,
+/// there is an attempt to apply an override state from the underlying data FeatureStore.
+@objc open class DynamicFeatureRegistry: FeatureRegistry {
+
+    /// A local store of the current dynamic features
+    private var dynamicFeatures: [String : AnyFeature] = [:]
+
+    override var features: [LabeledItem] {
+        let dynamicLabeledItems = dynamicFeatures.values.reduce(into: []) { (result, feature) in
```

Comment:
Why not use `map` here?

Reply:
Good question - I feel like there was a reason but whatever it was, it no longer relevant.